README.md

```diff
@@ -36,6 +36,12 @@ to [Visual Studio Code][VSC], rather than an emulation layer on top of it.
 
 For most [commands], the usage is the same as in [Kakoune]. However, the following changes have been made:
 
+### Selection Behavior
+
+Dance by default uses caret-based selections just like VSCode. This means a selection is anchored between two carets (i.e. positions between characters), and may be empty.
```

Comment:
Please keep lines in README under 120 characters (kinda arbitrary; it's just the soft limit I used when writing the rest of the document, and I'd like to keep _some_ sort of consistency).

Reply:
Wrapped. Feel free to either rebase merge or squash merge as you prefer.